### ISSUE TITLE
--try and --job don't get along

### ISSUE THREAD

**Issue by wpoely86** [hpcugent/easybuild-framework#1475]
If you want to use `--from-pr` (or just a directory with easyconfigs) and combine it with `--try-toolchain` and `--jobs`, it's not exactly user friendly. I use:
```
eb --disable-cleanup-tmpdir --tmpdir=$PWD/tmp --test-report-env-filter='' -r $PWD -l --try-toolchain=foss,2015b --job DL_POLY_Classic-1.9-intel-2015b-PLUMED-2.2.0.eb
```
This will fail with:
```
runpy ERROR Can't find path /vscmnt/gent_vulpix/_/user/home/gent/vsc403/vsc40307/files_pr2126/'2015b']" (at easybuild/framework/easyconfig/tools.py:626 in parse_easyconfigs)
```
which is probably the same bug as in #1438

We should fix this and make it more userfriendly.

**Comment by wpoely86** [comment]
This is what gets submitted by `--job`:
```
== 2015-11-16 10:21:02,502 runpy INFO Command line: --buildpath=/user/scratch/gent/vsc403/vsc40307/EB/golett/build --disable-cleanup-tmpdir --github-user=wpoely86 --installpath=/user/scratch/gent/vsc403/vsc40307/EB/golett/ --logtostdout --packagepath=/user/scratch/gent/vsc403/vsc40307/EB/golett/packages --prefix=/user/scratch/gent/vsc403/vsc40307/EB/golett/ --repositorypath=/user/scratch/gent/vsc403/vsc40307/EB/golett/ebfiles_repo --robot-paths=/user/home/gent/vsc403/vsc40307/files_pr2126: --sourcepath=/user/scratch/gent/vsc403/vsc40307/EB/sources:/apps/gent/source --test-report-env-filter=\"\" --testoutput=/vscmnt/gent_vulpix/_/user/home/gent/vsc403/vsc40307/files_pr2126/easybuild-build/PLUMED-2.2.0-foss-2015b --tmpdir=/user/home/gent/vsc403/vsc40307/files_pr2126/tmp --try-toolchain="['\"[\'foss\'', '']" /user/home/gent/vsc403/vsc40307/files_pr2126/tmp/eb-_zmfKs/tweaked_easyconfigs/PLUMED-2.2.0-foss-2015b.eb '2015b']"
```

**Comment by boegel** [comment]
duplicate of #1355?

**Comment by wpoely86** [comment]
Almost yes, #1438 and #1355 cover this.

**Comment by boegel** [comment]
proper fix here should be to filter `--try-*` in the submitted job, cfr. https://github.com/hpcugent/easybuild-framework/issues/1438#issuecomment-223916314

**Comment by boegel** [comment]
fixed in #1908, let's follow up there
